### PR TITLE
docs(PDEV-1173-docs): updating documentation

### PR DESCRIPTION
**Pull request by DatChive**
image

image

### CODE REVIEW

**Comment by bondricek** [comment]
getting clarification from Mike if its ok to change the structure or we need to keep it flat and just add the new properties

**Comment by bondricek** [comment]
Mike spoke with Coin and they are ok with the structural change in the response.  Also it was decided to add an optional accountGuid to both the get/put to being able to retrieve/update a specific account.

screen shot 2018-10-15 at 10 40 10 am
screen shot 2018-10-15 at 10 40 25 am

File: api/v10/examples/contact.res.json [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+  "clientContact": {
+    "firstName": "0TESTUSER0",
+    "lastName": "0SMITH0",
+    "addressLines": [
+      "350 N. ST. PAUL STREET"
+    ],
+    "city": "DALLAS",
+    "state": "TX",
+    "zip": "75201",
+    "emailAddress": "CELERYMAN@GMAIL.COM",
```
Comment (jamesdaniello) [inline api/v10/examples/contact.res.json]: image

File: api/v10/examples/contact.res.json [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+  },
+  "liquidNetWorth": {
+    "min": 100001,
+    "max": 200000
+  },
+  "investmentExperience": null,
+  "riskTolerance": null,
+  "investmentObjective": "INCOME",
+  "federalTaxBracketPercent": 8.0,
+  "descriptionOrTitle": "TEST USER TITLE"
```
Comment (jamesdaniello) [inline api/v10/examples/contact.res.json]: do we really want this included as a field? I thought we were just using it internally

Comment (bondricek) [reply api/v10/examples/contact.res.json]: i can remove it

**Pull request merged**